manifests/basic_ucp/ucp-armada.yaml

```diff
@@ -0,0 +1,505 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: helm-toolkit
+data:
+  chart_name: helm-toolkit
+  release: helm-toolkit
+  namespace: helm-toolkit
+  timeout: 100
+  values: {}
+  source:
+    type: git
+    location: git://git.openstack.org/openstack/openstack-helm
```

Comment:
I would recommend as a best practice, whenever dealing with a public repository to use https vs git, as https has the added advantage of server identity verification and https vs git generally will pass unmolested through more network filters.